package.json

```diff
@@ -4,9 +4,15 @@
   "scripts": {
     "dev": "vite",
     "build": "vite build",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "test": "vitest",
+    "coverage": "vitest --coverage"
   },
```

Comment:
Be sure to pull the latest changes on master, you'll have a much better experience.